### ISSUE TITLE
Spider: Northeastern Illinois University

### ISSUE THREAD

**Issue by pjsier** [City-Bureau/city-scrapers#728]
URL: https://www.neiu.edu/about/board-of-trustees/calendar-of-meetings
Doc URL: https://www.neiu.edu/about/board-of-trustees/board-meeting-materials
Spider Name: `chi_northeastern_il_university`
Agency Name: Northeastern Illinois University

Meeting title can default to "Board of Trustees" unless a specific committee is listed. We also want to scrape the docs here on the Doc URL. It should default to the main location, but check for asterisks which indicate alternate locations. Since it looks like the meeting time isn't anywhere on the page, it's fine to default to midnight and add a note about "See agenda for meeting time" in the `time_notes` field

See the [contribution guide](https://cityscrapers.org/docs/development/#contribute) for information on how to get started

**Comment by SaanikaKulkarni** [comment]
I'd like to work on this!

**Comment by pjsier** [comment]
@SaanikaKulkarni sounds great! Feel free to reach out with any questions

**Comment by SaanikaKulkarni** [comment]
Hi,
I keep getting an Index out of range error whenever I write a title_test. I tried printing the contents of 'parsed_items[]' but it's coming out to be empty. Am I doing something wrong with the file_response() method, because the city-scrapers-core documentation says it's to specify encoding for Windows while I'm working on a Mac? Thanks!

**Comment by pjsier** [comment]
@SaanikaKulkarni could you open a PR with what you have so far? It's hard to say without seeing your code, and that way we could troubleshoot

**Comment by rosemarymomoh** [comment]
hi @pjsier , doesn't look like there's been an update on this for a while now, mind if I take a stab at this?

**Comment by WhatAreYouSaying** [comment]
If this issue is unclaimed, I would like to work on that.

**Comment by pjsier** [comment]
@WhatAreYouSaying all yours!

**Comment by alderocas** [comment]
@pjsier I can work on this if you like

**Comment by pjsier** [comment]
Sure! Assigning you now

**Comment by alderocas** [comment]
Hi @pjsier,

What classification would you like to give meetings such as the 2018 board meeting: `Thursday, November 15 - All Committee Meetings followed by Board Meeting`? 

Is the title `Board of Trustees` unless a specific committee is listed still preferred?

**Comment by pjsier** [comment]
@alderocas Yeah I think "Board of Trustees" still works and the `BOARD` classification if there's a board meeting at all, thanks for clarifying!

**Comment by alderocas** [comment]
@pjsier Alright, I'll do that. Thank **you** for clarifying! :D

Have a great weekend!
